Microsoft.Toolkit.Uwp/Helpers/DispatcherHelper.cs

```diff
@@ -165,7 +165,16 @@ public static Task<T> AwaitableRunAsync<T>(this CoreDispatcher dispatcher, Func<
              {
                  try
                  {
-                     taskCompletionSource.SetResult(await function().ConfigureAwait(false));
+                     var awaitableResult = function();
+                     if (awaitableResult != null)
+                     {
+                         var result = await awaitableResult.ConfigureAwait(false);
+                         taskCompletionSource.SetResult(result);
```

Comment:
Will produced same result because result is the generic object.
if it is a value type it will not be possible for the caller to return a null and if it is a reference type I should do nothing becuase maybe the user wants this and even if I check default(referenceType)= null :)